api/src/pcapi/routes/external/users_subscription.py

```diff
@@ -136,3 +136,17 @@ def ubble_webhook_update_application_status(
     ubble_subscription_api.update_ubble_workflow(fraud_check, body.status)
 
     return ubble_validation.WebhookDummyReponse()
+
+
+@public_api.route("/webhooks/store_ubble_id_pictures", methods=["POST"])
```

Comment:
(détail / nommage)

Je propose de suivre le nommage des autres endpoints de type webhook : 

```suggestion
@public_api.route("/webhooks/ubble/store_id_pictures", methods=["POST"])
```